packages/dx-react-grid/docs/guides/grouping.md

```diff
@@ -73,8 +73,8 @@ You can also assign a function that returns a Boolean value depending on the `co
     />
     <LocalGrouping />
     <Table />
-    <TableHeaderRow showColumnWhenGrouped={columnName => columnName === 'city' || columnName === 'car'}/>
-    <TableGroupRow />
+    <TableHeaderRow />
+    <TableGroupRow showColumnWhenGrouped />
```

Comment:
Should we change the name to `showColumnsWhenGrouped`?

Reply:
Makes sense.

Reply:
@viterobk, can you help us?